### DISCUSSION TITLE
how’s my start gcode look

### DISCUSSION THREAD

**Discussion by 3d-not-2d** [mriscoc/Ender3V2S1#1079] [General]
I want to probe before every print and then it live adjust based on those values

With this as my start .gcode does it accomplish that or do I need to add/remove something?

    G92 E0
    G28
    G29 A
    C108
    G29 L0
    M500

**Comment by mriscoc** [comment]
Use this: https://github.com/mriscoc/Ender3V2S1/wiki/Slicer-G-code-Scripts
